### PR TITLE
update message

### PR DESCRIPTION
**Pull request by su-chang**

### CODE REVIEW

File: root/api/message.md [hunk 1]
```diff
@@ -19,22 +19,23 @@ All wechat messages will be encapsulated as a Message.
     * [.room\(\)](message.md#Message+room) ⇒ `Room` \| `null`
     * [~~.content\(\)~~](message.md#Message+content)
     * [.text\(\)](message.md#Message+text) ⇒ `string`
-    * [.say\(textOrContactOrFile, \[mention\]\)](message.md#Message+say) ⇒ `Promise.`
+    * [.say\(textOrContactOrFile, \[mention\]\)](message.md#Message+say) ⇒ `Promise <void>`
```
Comment (windmemory) [inline root/api/message.md]: This is not correct anymore.
The `mention` argument has been removed from `message.say` function.

File: root/api/message.md [hunk 2]
```diff
@@ -204,19 +205,19 @@ bot
...
-  const linkPayload = new UrlLnik({
-    description: 'Netty',
-    thumbnailUrl: 'http://mmbiz.qpic.cn/mmbiz_jpg/48MFTQpxichmmxEoXZ1w7eno72H2MQdx1WC6JiaVdYRmwAp4MCcQbctE2IE7jWqkWOlgMPqMBXVAdR1N46xEibvoQ/640?wx_fmt=jpeg&wxtype=jpeg&wxfrom=0',
-    title: 'Netty',
-    url: 'http://mp.weixin.qq.com/s?__biz=MzU2MDU3MzE1Mg==&mid=2247484375&idx=1&sn=5ee91b0a8607a1766b5212a23d3c9179&chksm=fc04bc58cb73354e798403bcc03e293149bb115a0755940e334c0fbe33d7c3b0b0797120a213&scene=0&xtrack=1#rd', 
-  })
-  await msg.say(linkPayload) 
-}
+  if (/^link$/i.test(m.text())) { 
+    const linkPayload = new UrlLink({
```
Comment (windmemory) [inline root/api/message.md]: The name of the variable should be the same with the Chinese version.

In the other one, this is `urlLink`.

File: root/api/message.md [hunk 3]
```diff
@@ -158,10 +158,11 @@ bot
   }
 })
 .start()
+```
 
-### message.say\(textOrContactOrFileOrUrlLink, \[mention\]\) ⇒ `Promise <void>`
+### message.say\(textOrContactOrFileOrUrlLink\) ⇒ `Promise <void>`
 
-Reply a Text, Media File or Link message to the sender.
+Reply a Text,Contact Card, Media File or Link message to the sender.
```
Comment (windmemory) [inline root/api/message.md]: need one space before the `Contact Card`

**Comment by lijiarui** [comment]
@su-chang  Please resolve all of the conversations, thanks!

File: root/api/message.md (hunk 1 above)
Comment (su-chang) [reply root/api/message.md]: Done

File: root/api/message.md (hunk 2 above)
Comment (su-chang) [reply root/api/message.md]: Done, both is `urlLink` now.

File: root/api/message.md (hunk 3 above)
Comment (su-chang) [reply root/api/message.md]: Done, thx. I'll be more careful about that.

File: root/api/message.md (hunk 1 above)
Comment (windmemory) [reply root/api/message.md]: the mention argument is still there.

Comment (su-chang) [reply root/api/message.md]: Removed.

**Pull request merged**